website/blog/2023-01-04-databend-weekly-75.md

```diff
@@ -0,0 +1,151 @@
+
+## What's New
+
+Check out what we've done this week to make Databend even better for you.
+
+### Features & Improvements :sparkles:
+
+**Format**
+
+- basic output format JSON ([#9447](https://github.com/datafuselabs/databend/pull/9447))
+
+**Query**
+
+- check connection params ([#9437](https://github.com/datafuselabs/databend/pull/9437))
+- add `max_query_row_nums` ([#9406](https://github.com/datafuselabs/databend/pull/9406))
+
+**Storage**
+
+- support prewhere in hive ([#9427](https://github.com/datafuselabs/databend/pull/9427))
+- add generic cache trait for different object reader ([#9436](https://github.com/datafuselabs/databend/pull/9436))
+- add metrics for new cache ([#9445](https://github.com/datafuselabs/databend/pull/9445))
+
+**New Expression**
+
+- migrate hash func to func-v2 ([#9402](https://github.com/datafuselabs/databend/pull/9402))
+
+**Sqllogictest**
+
+- run all tests in parallel ([#9400](https://github.com/datafuselabs/databend/pull/9400))
+
+### Code Refactoring :tada:
+
+**Storge**
+
+- add `to_bytes` and `from_bytes` for `CachedObject` ([#9439](https://github.com/datafuselabs/databend/pull/9439))
+- better table-meta and parquet reader function ([#9434](https://github.com/datafuselabs/databend/pull/9434))
+- convert fuse_snapshot unit tests to sqlloigc test ([#9428](https://github.com/datafuselabs/databend/pull/9428))
+
+### Bug Fixes :wrench:
+
+**Format**
+
+- atch unwind when read split ([#9420](https://github.com/datafuselabs/databend/pull/9420))
+
+**User**
+
+- lazy load JWKS ([#9446](https://github.com/datafuselabs/databend/pull/9446))
+
+**Planner**
+
+- create Stage URL's path should ends with `/` ([#9450](https://github.com/datafuselabs/databend/pull/9450))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Databend 2022 Recap
+
+Let's look back and see how Databend did in 2022.
+
+- Open Source: got 2,000+ stars, merged 2,400+ PRs, solved 1,900 issues.
+- Brand new data warehouse, which is planned as Lakehouse.
+- Rigorous testing, including SQL Logic Tests, SQLancer and <https://perf.databend.rs> .
+- Growing with users and actively expanding our ecosystem, Databend and its surrounding projects have been recognised by a number of teams, including Kuaishou and SAP.
+- Built on top of Databend, we have launched Databend Cloud, the next generation big data analytics platform.
+
+We wish everyone a Happy New Year and look forward to engaging with you.
+
+**Learn More**
+
+- [Blog | Databend 2022 Recap](https://databend.rs/blog/2022-12-31-databend-2022-recap)
+
+
+### Databend 2023 Roadmap
+
+As the new year approaches, Databend is also actively planning its roadmap for 2023.
+
+We will continue to polish Planner and move forward with caching of data and queries, and we are also trying to address storage and query issues for PB-level data volumes.
```

Comment:
We will continue to polish the Planner and work on data and query caching. Enhancing storage and query issues for PB-level data volumes is also on our list.